system-test/system-test.sh

```diff
@@ -38,7 +38,7 @@ echo "Starting Spark application"
 ./system-test/spark-logging &> system-test/log_files/spark_example.log
 
 echo "Curling the Solr API"
-query_response=$(curl -i -o - --silent -X GET -u ${SOLR_USR}:${SOLR_PWD} "http://master3.valhalla.phdata.io:8983/solr/logging-pulse-test_latest/select?q=*%3A*&wt=json&indent=true")
+query_response=$(curl -i -o - --silent -X GET -u ${SOLR_USR}:${SOLR_PWD} "http://master3.${ZK_HOST}:8983/solr/logging-pulse-test_latest/select?q=*%3A*&wt=json&indent=true")
```

Comment:
Do we need to document in this script that the `ZK_HOST` param is required?

Reply:
This is outdated. But added ZK_HOST and other required variables to env.sh.template